java/code/src/com/redhat/rhn/frontend/action/LoginHelper.java

```diff
@@ -44,23 +44,18 @@
 import org.apache.commons.lang3.StringUtils;
 import org.apache.commons.lang3.time.StopWatch;
 import org.apache.log4j.Logger;
-import org.apache.struts.action.ActionErrors;
-import org.apache.struts.action.ActionMessage;
-import org.apache.struts.action.ActionMessages;
 
-import java.io.IOException;
 import java.io.UnsupportedEncodingException;
 import java.util.ArrayList;
 import java.util.Date;
 import java.util.HashMap;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
-
+import javax.security.auth.login.LoginException;
```

Comment:
This does not matter so much and you can decide to ignore it, but the original [upstream import order configuration](https://github.com/uyuni-project/uyuni/blob/master/java/conf/eclipse/rhn.importorder) would keep `java` imports separately from `javax`.

Reply:
OK, I updated the IntelliJ wiki rules to reflect this: https://github.com/uyuni-project/uyuni/wiki/IntelliJ-IDEA-specific-development-instructions/_compare/8da7f60de53dcd4f17b9428e3ced6ec8fe468fdb